code/modules/vehicles/unmanned/unmanned_droid.dm

```diff
@@ -29,31 +29,33 @@
 		STOP_PROCESSING(SSslowprocess, src)
 		user.clear_fullscreen("machine", 5)
 
-/obj/vehicle/unmanned/droid/welder_act(mob/living/user, obj/item/I)
+/obj/vehicle/unmanned/droid/welder_act(mob/living/user, obj/item/I)	//used to heal the droid
 	if(user.do_actions)
 		balloon_alert(user, "Already busy!")
 		return FALSE
 	if(obj_integrity >= max_integrity)
+		balloon_alert(user, "[src] doesn't need repairs")
 		return TRUE
 	if(user.skills.getRating("engineer") < SKILL_ENGINEER_ENGI)
-		balloon_alert_to_viewers("[user] fumbles repairs", ignored_mobs = user)
-		balloon_alert(user, "You fumble repair")
+		balloon_alert_to_viewers("[user] tries to repair [src]" , ignored_mobs = user)
+		balloon_alert(user, "You try to repair [src]")
```

Comment:
Do not use [src] here, ballon alerts must be as short as possible and targets are already known because src is where the aler comes from